api/v1alpha3/vspherevm_types.go

```diff
@@ -44,6 +44,9 @@ type VSphereVMSpec struct {
 	// this CRD as unstructured data.
 	// +optional
 	BiosUUID string `json:"biosUUID,omitempty"`
+
+	// EnableSecureVCenterTransport is a flag that disables TLS peer verification.
+	EnableSecureVCenterTransport bool `json:"insecure,omitempty"`
```

Comment:
Hi @alexander-demichev - sorry to ask you this again, but after some discussion we feel like having `insecure` here (i.e. same name as `VSphereCluster` is easier. could you revert the rename ?

Reply:
:+1: